src/components/ec-input-field/ec-input-field.vue

```diff
@@ -114,13 +114,13 @@ $ec-input-background-disabled: $level-7-backgrounds !default;
   }
 
   &__label {
-    @include caption-text-bold;
+    @include input-text-bold;
```

Comment:
I think @mcibique had some comments in the previous PR

Reply:
The previous comment was twofold. 
1) Use pre-existing "input-label" typography
2) Remove name bold from mixin

I didn't find any pre-existing "input-label" but we did have @caption-text-bold mixin that the "ec-input-field" component was using. Even so though this mixing had a mistake as it was using the wrong color.

I removed it completely and replaced with a new "input-label" label mixin that resolves @mcibique comment and fixes both components.